### PR TITLE
Added option to use legacy Reddit favicon

### PR DESCRIPTION
**Pull request by corylulu**
Could have made it not dependant on `showUnreadCountInFavicon`, but it would have made it uglier and I don't think it matters much.

### CODE REVIEW

File: lib/modules/orangered.js [hunk 1]
```diff
@@ -216,6 +223,10 @@ const setupFaviconBadge = _.once(() => {
 	for (const f of favicons) {
 		if (f !== selectedFavicon) f.remove();
 	}
+	// browser should auto resolve to this if all get removed, but just in case, setting it explicitly
+	if (module.options.faviconUseLegacy.value) {
+		selectedFavicon.href = '//www.reddit.com/favicon.ico';
```
Comment (andytuba) [inline lib/modules/orangered.js]: Let's use `redditstatic.com` instead of `reddit.com`

File: locales/locales/en.json [hunk 2]
```diff
@@ -4360,5 +4360,11 @@
 	"submittedToSubreddit": {
 		"message": "to",
 		"description": "[post/comment submitted...] to $subreddit"
+	},
+	"orangeredFaviconUseLegacyTitle": {
+		"message": "Use Legacy Favicon"
+	},
+	"orangeredFaviconUseLegacyDesc": {
+		"message": "If you don't like the new orange Reddit favicon, revert to using the old Reddit icon instead."
```
Comment (andytuba) [inline locales/locales/en.json]: orangered*

Comment (corylulu) [reply locales/locales/en.json]: Is it all fine now?

**Comment by erikdesjardins** [comment]
Build failure looks spurious

**Pull request merged**